.github/workflows/build_and_test.yml

```diff
@@ -7,16 +7,18 @@ on:
   schedule:
     # Run every hour. This helps detect flakiness,
     # and broken external dependencies.
+    # Only triggered for the master branch
```

Comment:
Triggered on the master branch, but build_and_test are scheduled for both master and release_latest right (using vcs_url)? This might be misleading.

Reply:
The release_latest branch is never triggered. The vcs file is to ensure we test that branch. Will update the comment.